crates/bevy_dev_tools/src/fps_overlay.rs

```diff
@@ -96,3 +111,21 @@ fn customize_text(
         }
     }
 }
+
+fn change_visibility(
+    mut query: Query<&mut Visibility, With<FpsText>>,
```

Comment:
Can't this cause conflict error everywhere when the user uses mut Visibility? Or will not cause conflict because Query know when archetypes cross each other?

Reply:
From what I know this shouldn't cause any issues.

Reply:
Yeah, this will generally run in parallel. Those checks are based on "factual compatibility": they check the list of actual archetypes.